dan_layer/transaction_manifest/src/parser.rs

```diff
@@ -324,6 +324,7 @@ fn macro_call(mac: &Ident, tokens: TokenStream) -> Result<ManifestIntent, syn::E
             level: LogLevel::Error,
             message: parse2::<LitStr>(tokens)?.value(),
         })),
+        "store" => todo!("Implement me later"),
```

Comment:
Is this needed? What is the intention?

Reply:
It was supposed to be removed, but missed it. Thanks for catching it :)